lk2nd/device/dts/msm8952/msm8937-nokia-ple.dts

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8937 0x2000>,
+		      <QCOM_ID_MSM8937 0x3000>;
+	qcom,board-id = <0x9B 0>;
+};
+
+&lk2nd {
+	ple {
+		model = "HMD Global Nokia 6 (ple)";
+		compatible = "nokia,ple";
+		qcom,msm-id = <QCOM_ID_MSM8937 0x2000>,
+			      <QCOM_ID_MSM8937 0x3000>;
```

Comment:
This override shouldn't change anything unless the original bootloader is completely bogus, and lk2nd will always only use the first value in the list anyway.

Reply:
Gotcha.